concent_api/core/models.py

```diff
@@ -11,7 +11,7 @@ class Message(Model):
     type        = PositiveSmallIntegerField()
     timestamp   = DateTimeField()
     data        = BinaryField()
-    task_id     = IntegerField(null = True)
+    task_id     = CharField(max_length = 255, null = True)
```

Comment:
Could you check if Golem code has any limit on the length of task IDs?

I think that if we impose a limit here, we should have some kind of validation of that limit in golem-messages. And perhaps a constant that defined the maximum ID length.

Reply:
I have checked in docs and code and couldn't find any.

Reply:
OK. But we still need a validation.